src/components/az-list/AZList.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import "./AZList.scss";
+
+import { testIds } from "../../constants";
+import { useAppPaths } from "../../hooks/routing";
+import { useStateValue } from "../../store/store";
+import { i18n } from "../../utils/i18n";
+
+const AZList = () => {
+    const { HomePath } = useAppPaths();
+    const [{ language }] = useStateValue();
+    const baseURL = `${HomePath()}?expand=`;
+    const listArray = 'abcdefghijklmnopqrstuvwxyz#'.split('');
+    return (
+        <div className="list-container" data-testid={testIds.AZ_LIST}>
+            <div>
+                {i18n.browse[language]}:
+                <ul>
+                    { listArray.map( (item, i) => {
+                       const itemLink = item === '#' ? '%23' : item.toUpperCase();
+                       const label = item.toUpperCase();
+                       return (
+                           <li key={i} value={i+1}>
+                               <Link to={`${baseURL}${itemLink}`}>{label}</Link>
```

Comment:
`<Link to=` should be
`<Link to={Expand({ expandChar: itemLink})}`

Reply:
Done!